contract_agent_interface/partner.py

```diff
@@ -90,7 +90,7 @@ def _get_analytic_lines(self, cr, uid, ids, field_name,
             'Payments'),
         'all_bank_ids': fields.one2many(
             'res.partner.bank', 'partner_id', 'Banks',
-            domain=['|', ('active', '=', False), ('active', '=', True)],
```

Comment:
The original value is the right one here. The reason there is this domain is unless you have a test for the field "active" in your domain, inactive entries will never be returned by the ORM.

In this case we want to be able, through the all_bank_ids field, to view all bank accounts for a partner. The rest of the code can keep on using bank_ids and never see the inactive ones.